base-notebook/start.sh

```diff
@@ -16,6 +16,9 @@ if [ $UID == 0 ] ; then
                 chown -R $NB_UID "$d"
             fi
         done
+    elif [[ -d /home/$NB_USER/work && $(stat -c %U /home/$NB_USER/work) != "$NB_USER" ]]; then
```

Comment:
In this case, I would do both. I don't know how you use this docker image, but me, I just mount a Marathon volume with `/home/$NB_USER/work` (we are migrating to kubernetes, I guess it is the same). So actually we should chmod the mount point